cub/test/catch2_test_launch_helper.h

```diff
@@ -76,6 +76,25 @@
 #  error Test file should contain %PARAM% TEST_LAUNCH lid 0:1
 #endif
 
+#define DECLARE_INVOCABLE(API, WRAPPED_API_NAME)                                          \
+  struct WRAPPED_API_NAME##_invocable_t                                                   \
```

Comment:
I was wondering as well whether the previous version was avoiding an ODR violations, but the invocables are exactly the same for all values of `TEST_LAUNCH` and the launch wrapper itself is `static`, so it can appear in different TUs with different definitions.

Reply:
> If you want to simplify this file further, we can get rid of all the `#define launch host_side_api_launch` macro and rename corresponding functions `host_side_api_launch` -> `launch`.

Applied.